### ISSUE TITLE
[508-defect-4] CST: Fix voiceover header text span bug

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#41822]
## Description

[VoiceOver headings span bug](https://user-images.githubusercontent.com/14154792/169866051-475aed9b-a6ed-4be3-bb32-a31eb6b169d9.PNG) splits up headings and requires span role="text" fix

Related: https://github.com/department-of-veterans-affairs/va.gov-team/issues/41447 (Under "Consider" fix)

!["Claim for" text separated from remaining header text](https://user-images.githubusercontent.com/14154792/169866051-475aed9b-a6ed-4be3-bb32-a31eb6b169d9.PNG)

## Specs:
- Device: iPhone
- Browser: Safari
- Test User (if applicable): vetsgov+test228@id.me (Colder Polarbear)

## Steps to reproduce
- URL: https://staging.va.gov/track-claims/your-claims
- Test User (if applicable): vetsgov+test228@id.me (Colder Polarbear)
- **NOTE**: This is a bug with VoiceOver, so you will need to enable that in Settings > Accessibility > VoiceOver on your iPhone
---
* Login and navigate to the URL indicated above
* You should be able to scroll down the page until you see content similar to the screenshot above

## Tasks

- [ ] Header text within card should not be broken up for screenreaders
- [ ] Add tests (as needed)

## Definition of done

- [ ] All tasks complete
- [ ] All tests passing
